### ISSUE TITLE
Understanding Accessible Authentication 3.3.7

### ISSUE THREAD

**Issue by littlea11ywidget** [w3c/wcag#1313]
Hi, Some feedback on the 'Intent' under  'Understanding Accessible Authentication 3.3.7'  

May I suggest that the content which starts with 'Cognitive Function tests are known to be problematic...' come before 'Most sites rely on user names and passwords for logging in' ? After reading the Success Criteria, I was ready to dig into more information on cognitive function tests but suddenly finding myself reading about user id's and passwords made me feel like I misread something. Moving the part about user names and passwords to be after will also position it closely to the note, which unless I actually am missing something feels appropriate because it has a closer relationship.

**Comment by alastc** [comment]
Hi @littlea11ywidget,

I see what you mean, but I wouldn't want to loose the scope of the SC either, it is within the context of authentication. I.e. it is only about cognitive function tests _for_ authentication.

How about adjusting that second paragraph of the [understanding doc](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication.html) so it flows better, starting off like this:
> The general case of remembering a password is a [link]cognitive function test[/link], which are known to be problematic for many people with cognitive disabilities....

You can see the ['diff' of that change](https://github.com/w3c/wcag/pull/1412).
